frontstage_api/error_handlers.py

```diff
@@ -1,25 +1,30 @@
 import logging
 
+from flask import jsonify
 from structlog import wrap_logger
 
-from frontstage_api import api
-from frontstage_api.exceptions.exceptions import FailedRequest, InvalidRequestMethod, NoJWTError, UnexpectedStatusCode
+from frontstage_api import api, app
+from frontstage_api.exceptions.exceptions import ApiError, InvalidRequestMethod, NoJWTError
 
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-@api.errorhandler(FailedRequest)
-def failed_request(error):
-    message_json = {
-        'message': 'External request failed',
-        'url': error.url,
-        'method': error.method,
-        'exception': error.exception
+@app.errorhandler(ApiError)
+@api.errorhandler(ApiError)
```

Comment:
Would http://flask.pocoo.org/docs/0.12/api/#flask.Blueprint.app_errorhandler cover both the scenarios?

Reply:
`api.errorhandler` is the intended solution but I think there's a problem with it in Flask-Restplus, I'll try just `app.errorhandler` and see if it has the same effect

Reply:
yeah that doesn't work, I believe it's a known issue in restplus currently there's a couple of github issues about it. This is where I found this solution
https://github.com/noirbizarre/flask-restplus/issues/334